src/rust/grapl-web-ui/src/config.rs

```diff
@@ -13,14 +13,15 @@ pub(crate) const SESSION_TOKEN: &'static str = "SESSION_TOKEN";
 pub(crate) const SESSION_TOKEN_LENGTH: usize = 32;
 pub(crate) const SESSION_EXPIRATION_TIMEOUT_DAYS: i64 = 1;
 
-// Try getting an environment variable and create ConfigError::MissingEnvironmentVariable if
-// unsuccessful. The benefit of this over just std::env::VarError is the environment variable will
-// be Display'd as well.
-macro_rules! env_var {
-    ($name:expr) => {
-        std::env::var($name)
-            .map_err(|var_err| ConfigError::MissingEnvironmentVariable($name, var_err))
-    };
+fn get_env_var(name: &'static str) -> Result<String, ConfigError> {
+    std::env::var(name).map_err(|source| ConfigError::MissingEnvironmentVariable {
+        variable_name: name,
+        source,
+    })
+}
```

Comment:
Thank you for making this a function, rather than a macro 🙏

Reply:
I think I made those macros and I couldn't remember why I did that.